js/src/views/Signer/containers/RequestsPage/requestsPage.js

```diff
@@ -85,7 +88,9 @@ class RequestsPage extends Component {
       >
         <TxList
           address=''
+          blockNumber={ blockNumber }
           hashes={ localHashes }
+          newError={ newError }
```

Comment:
Ok, it's not inline with some other functions in the same file though, we map redux actions to component events like this:
```js
onConfirm={ startConfirmRequest }
onReject={ startRejectRequest }
```